cvmfs/swissknife_graft.cc

```diff
@@ -0,0 +1,249 @@
+/**
+ * This file is part of the CernVM File System.
+ *
+ * Process a set of input files and create appropriate graft files.
+ */
+
+#include <fcntl.h>
+#include <unistd.h>
+#include <sys/stat.h>
+
+#include <cstdio>
+#include <iostream>
+#include <vector>
+#include <sstream>
+
+#include "compression.h"
+#include "fs_traversal.h"
+#include "hash.h"
+
+#include "swissknife_graft.h"
+
+#define CHUNK_SIZE 16*1024
```

Comment:
This would be a chunk for every 16kB, right?  I think that's very small.  Native files chunk as of a few MB.